python/ray/_private/worker.py

```diff
@@ -2829,7 +2837,14 @@ def _make_remote(function_or_class, options):
     if not function_or_class.__module__:
         function_or_class.__module__ = "global"
 
+    actor_class: bool = options.pop("actor_class", False)
     if inspect.isfunction(function_or_class) or is_cython(function_or_class):
+        if actor_class:
+            raise TypeError(
+                "The @ray.remote decorator must be applied "
+                "to a class when actor_class=True"
```

Comment:
This is backward incompatible.

Reply:
Can you accept the original proposal and add a new `actor_class` decorator？

Reply:
Regardless, these modifications theoretically do not affect the previous code. If the user accepts this new style remote call, it implies that they accept updating all ray versions to the latest one.